Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -705,6 +720,22 @@ private static String createAssertionId() {
         return ID_PREFIX.concat(String.valueOf(UUID.randomUUID())).replaceAll("-", "");
     }
 
+    protected boolean isConditionsDefaultValueEnabled() {
+        // if not provided or invalid return true else false
+        try {
+            final String ConditionsDefaultValueEnabled = PropertyAccessor.getInstance()
+                .getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.ENABLE_CONDITIONS_DEFAULT_VALUE);
```

Comment:
The reason I used .getProperty was the AuthzDecisionConditions property was defined in the same way (See Below).
However, if I were to change one I should change the other.
    protected boolean isAuthDEvidenceConditionsDefaultValueEnabled() {
        // if not provided or invalid return true else false
        try {
            final String authDEvidenceConditionsDefaultValueEnabled = PropertyAccessor.getInstance().getProperty(
                NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.ENABLE_AUTH_DEC_EVIDENCE_CONDITIONS_DEFAULT_VALUE);